Source/CarthageKit/Xcode.swift

```diff
@@ -866,6 +866,29 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 				fatalError("SDK count \(platform.SDKs.count) for platform \(platform) is not supported")
 			}
 		}
+
+	return builtProduct
```

Comment:
Can you remove this variable and just add the `|> …` to the end of the chain of operations?